docs/modules/ROOT/pages/cops_rails.adoc

```diff
@@ -668,6 +668,45 @@ end
 | Array
 |===
 
+== Rails/CustomValidator
+
+|===
+| Enabled by default | Safe | Supports autocorrection | VersionAdded | VersionChanged
+
+| Pending
+| Yes
+| No
+| 2.7
+| -
+|===
+
+This cop suggests extracting custom validator classes from format validations.
+
+=== Examples
+
+[source,ruby]
+----
+# bad
+class User < ApplicationRecord
+  validates :email, format: { with: /@/ }
```

Comment:
I wonder if it would be better to illustrate this with a more involved example, where we can show that extracting a validator leads to a much cleaner model?

Reply:
Yes, I probably have to use an example from style guide link from the beginning. Will do that.